source/jsonnet/northern-ireland/census_household.jsonnet

```diff
@@ -53,6 +53,7 @@ local frequency_irish = import 'individual/blocks/identity-and-health/frequency_
 local frequency_ulster_scots = import 'individual/blocks/identity-and-health/frequency_ulster_scots.jsonnet';
 local health = import 'individual/blocks/identity-and-health/health.jsonnet';
 local health_conditions_or_illnesses_limitations = import 'individual/blocks/identity-and-health/health_conditions_or_illnesses_limitations.jsonnet';
+local level_of_spoken_english = import 'individual/blocks/identity-and-health/level_of_spoken_english.jsonnet';
```

Comment:
I can't see this change specified in the card or the referenced tab on the Google sheet. Is there another tab or doc that has been used for changes?

Reply:
I changed some filenames because there was a little inconsistency with the ids we use, same was done to _visitor_date_of_birth_ and _visitor_sex_.